lib/maths/CBoostedTreeImpl.cc

```diff
@@ -1463,8 +1494,6 @@ bool CBoostedTreeImpl::acceptRestoreTraverser(core::CStateRestoreTraverser& trav
                                              m_MaximumNumberTrees, traverser))
         RESTORE(TRAINING_ROW_MASKS_TAG,
                 core::CPersistUtils::restore(TRAINING_ROW_MASKS_TAG, m_TrainingRowMasks, traverser))
-        RESTORE(TRAINING_PROGRESS_TAG,
```

Comment:
Will we safely ignore this tag in model states from previous versions?

Reply:
Yes if a tag turns up we don't know about we just ignore in the restore code.

Reply:
See [3d0b42e](https://github.com/elastic/ml-cpp/pull/1179/commits/3d0b42e404747cdfc195c609f369118de806fafc).